connect/src/lib/utils.test.js

```diff
@@ -26,6 +26,98 @@ describe('utils', () => {
     })
   })
 
+  describe('serializeCron', () => {
+    describe('should throw error if incorrect cron data', () => {
+      describe('should throw error if interval is incorrect', () => {
+        test('should throw error if interval is non-string', () => {
+          let errorCaught = false
+          const cronWrongInterval = {
+            interval: 10,
+            tags: [
+              { name: 'foo', value: 'bar' }
+            ]
+          }
+          try {
+            serializeCron(cronWrongInterval)
+          } catch (e) {
+            errorCaught = true
+            assert.equal(e.message, 'Encountered Error serializing cron: invalid interval')
+          }
+          assert.ok(errorCaught)
+        })
```

Comment:
Could you add spaces in between the tests? Just makes them a bit easier to read.